ResearchKit/Common/ORKImageCaptureView.m

```diff
@@ -119,6 +115,7 @@ - (void)orientationDidChange {
     }
     [_previewView setVideoOrientation:orientation];
     [self.delegate videoOrientationDidChange:orientation];
+    [self setNeedsUpdateConstraints];
```

Comment:
So, out of this whole change, this seems to be the *only* functional change. Is that also your understanding?